iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -91,7 +91,14 @@ export class NativeLibrary {
   public static get nativeLib() { return this.load(); }
   public static load() {
     if (!this._nativeLib) {
-      this._nativeLib = require(`./${NativeLibrary.archName}/${NativeLibrary.nodeAddonName}`) as typeof IModelJsNative; // eslint-disable-line @typescript-eslint/no-var-requires
+      try {
+        this._nativeLib = require(`./${NativeLibrary.archName}/${NativeLibrary.nodeAddonName}`) as typeof IModelJsNative; // eslint-disable-line @typescript-eslint/no-var-requires
+      } catch (err: any) {
+        err.message += "\nThis error may occur when trying to run an iTwin.js backend without"
+          + " having installed the prerequisites. See the following link for all prerequisites:"

```

Comment:
```suggestion
          + "\nhaving installed the prerequisites. See the following link for all prerequisites:"

```

Reply:
no worries, just a nit. i assume you broke this error msg string up for formatting